lua/cmp/view.lua

```diff
@@ -189,10 +192,14 @@ view._get_entries_view = function(self)
     end)
     return self.native_entries_view
   else
-    self.custom_entries_view.event:on('change', function()
+    local v = self.custom_entries_view
+    if vim.tbl_contains({ '/', '?' }, vim.fn.getcmdtype()) and c.experimental.horizontal_search then
+      v = self.statusline_entries_view
+    end
+    v.event:on('change', function()
       self:on_entry_change()
     end)
-    return self.custom_entries_view
+    return v
```

Comment:
```lua
cmp.setup {
  appearance = {
    entries_view = 'custom' | 'statusline' | 'native', -- the `native` will be removed in the future.
  }
}
```

I'm afraid the `appearance` and `entries_view` are good wording or not. (I'm always afraid of such wording decisions...

Reply:
How about `display.layout` or `window.layout`?

Reply:
We already have `cmp.setup.cmdline`.
You can set appearance option for only cmdline-mode.

Reply:
Ok. So maybe we we can add the appearance config option both in general and in cmdline mode.
Makes sense?

Reply:
@tzachar Yes. I think this feature is needless in insert-mode. But it's useful to keep the consistency.

What do you think about this?
```
cmp.setup {
  view = {
    entries = 'custom' | 'statusline' | 'native',
  }
}
```

Reply:
Hm... `display` / `appearance` / `layout` / `window` / `view` / `ui`...

Reply:
I think we can merge this after solve this.

Reply:
OK. I decided.
I'll organize the configuration options after merging this PR.